src/main.cpp

```diff
@@ -88,6 +88,9 @@ enum Options {
   DEBUG,
   DRY_RUN,
 };
+
+constexpr auto FULL_SEARCH = "*:*";
```

Comment:
Nit: you can declare this in the scope where it is used. (You can keep it at the top if used broadly within the file, but I think the reasonable principle here is "smallest scope possible", e.g. if it was specific to a class you would keep it in the class, etc.

Reply:
Note: it's used in different functions in main, so I think file level is appropriate here.